src/makeRobot.js

```diff
@@ -38,7 +38,73 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get info() {
+      return `name: ${this.name},`
+      + ` chip version: ${this.version},`
+      + ` wheels: ${this.wheels}`;
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    goForward(value = 1) {
+      if (value < 0) {
+        this.coords.y += 0;
+
+        return this;
+      } else {
+        this.coords.y += value;
+
+        return this;
+      }
+    },
+    goBack(value = 1) {
+      if (value < 0) {
+        this.coords.y -= 0;
+
+        return this;
+      } else {
+        this.coords.y -= value;
+
+        return this;
+      }
+    },
```

Comment:
You can write it much shorter and without "else" if you invert value check
Please apply to other functions as well
```suggestion
    goBack(value = 1) {
      if (value > 0) {
        this.coords.y -= value;
      }
      
      return this;
    },
```